placeSpecificContent/hq/courtyard/controlPanel/controlPanel_server.js

```diff
@@ -0,0 +1,406 @@
+//
+// controlPanel_server.js
+// 
+// Created by Rebecca Stankus on 07/16/2019
+// Copyright High Fidelity 2019
+//
+// Licensed under the Apache 2.0 License
+// See accompanying license file or http://apache.org/
+//
+
+(function() {
+    var HALF = 0.5;
+
+    var DATA_RETRIEVAL_TIMEOUT_MS = 500;
+    var PRELOAD_DATA_RETRIEVAL_MAX_ATTEMPTS = 10;
+
+    var DONE_SOUND = SoundCache.getSound(Script.resolvePath("resources/sounds/DONE_SOUND.mp3?1"));
```

Comment:
I think we are trying to get away from cache busting so much

Reply:
OH, yeah, I just forgot to remove that for CR. Thanks!